### PR TITLE
AAP-734: Change all IDS to IDPS

### PR DESCRIPTION
**Pull request by lmalivert**
Changed all instances of "IDS" to "IDPS"

- Link to Jira: https://issues.redhat.com/browse/AAP-734

- Link to google doc: https://docs.google.com/document/d/1VU0BpbM8RUvchmYmjYdZslkgtvM1BJZV6ZT4Cbrklig/edit#

### CODE REVIEW

File: downstream/assemblies/security/assembly-idps.adoc [hunk 1]
```diff
@@ -16,25 +16,25 @@ ifdef::context[:parent-context: {context}]
...
-= Automating Network Intrusion Detection System (IDS) Rules with Ansible
+= Automating Network Intrusion Detection and Prevention Systems (IDPS) with Ansible
 
-:context: instrusion-detection-system
+:context: idps
 
 [role="_abstract"]
 
-You can use Ansible to automate your Intrusion Detection System (IDS). For the purpose of this guide, we use Snort as the IDS. Use Ansible automation hub to consume content collections, such as tasks, roles, and modules to create automated workflows.
+You can use Ansible to automate your Intrusion Detection and Prevention Systems (IDPS). For the purpose of this guide, we use Snort as the IDS. Use Ansible automation hub to consume content collections, such as tasks, roles, and modules to create automated workflows.
```
Comment (craig-br) [inline downstream/assemblies/security/assembly-idps.adoc:27]: `You can use Ansible to automate your Intrusion Detection and Prevention Systems (IDPS). For the purpose of this guide, we use Snort as the IDS.`

to 

`You can use Ansible to automate your Intrusion Detection and Prevention System (IDPS). For the purpose of this guide, we use Snort as the IDPS.`

**Review by craig-br — CHANGES_REQUESTED** [review]

**Comment by lmalivert** [comment]
Thanks for catching that @craig-br. Please review.

**Comment by craig-br** [comment]
@lmalivert . It's ready to merge

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**